scripts/scr_ui_manage/scr_ui_manage.gml

```diff
@@ -849,130 +857,142 @@ function scr_ui_manage() {
 	        }
 	        if (man[sel]="vehicle"){
 	            if (ma_lid[sel]=0) then draw_text_transformed(xx+430,yy+66,string(temp2),1,1,0);
 	        }*/
-        
+
 	        /*draw_set_color(38144);// Equipment
 	        if (string_count("Artifact",temp5)>0) then draw_set_color(881503);
 	        draw_text_transformed(xx+573,yy+66,string(temp5),1,1,0);draw_set_color(38144);*/
-        
+
 	        if (man[sel]="man"){var x5;x5=0;
 	            draw_set_color(c_gray);
 	            if (ar_ar=1) then draw_set_color(c_gray);
 	            if (ar_ar=2) then draw_set_color(881503);
 	            draw_text(xx+573,yy+66,string_hash_to_newline(string(ma_ar)));
-            
+
 	            x5+=string_width(string_hash_to_newline(ma_ar))+15;
 	            draw_set_color(c_gray);
 	            if (ar_mb=1) then draw_set_color(c_gray);
 	            if (ar_mb=2) then draw_set_color(881503);
 	            draw_text(xx+573+x5,yy+66,string_hash_to_newline(string(ma_mb)));
-            
+
 	            x5+=string_width(string_hash_to_newline(ma_mb))+15;
 	            draw_set_color(c_gray);
 	            if (ar_ge=1) then draw_set_color(c_gray);
 	            if (ar_ge=2) then draw_set_color(881503);
 	            draw_text(xx+573+x5,yy+66,string_hash_to_newline(string(ma_ge)));
-            
+
 	            x5+=string_width(string_hash_to_newline(ma_ge))+15;
 	            draw_set_color(c_gray);
 	            if (ar_we1=1) then draw_set_color(c_gray);
 	            if (ar_we1=2) then draw_set_color(881503);
 	            draw_text(xx+573+x5,yy+66,string_hash_to_newline(string(ma_we1)));
-            
+
 	            x5+=string_width(string_hash_to_newline(ma_we1))+15;
 	            draw_set_color(c_gray);
 	            if (ar_we2=1) then draw_set_color(c_gray);
 	            if (ar_we2=2) then draw_set_color(881503);
 	            draw_text(xx+573+x5,yy+66,string_hash_to_newline(string(ma_we2)));
 	        }
-        
+
 	        if (man[sel]!="man"){var x5;x5=0;
-        
-	            draw_set_color(c_gray);
-	            if (ar_ar=1) then draw_set_color(c_gray);
-	            if (ar_ar=2) then draw_set_color(881503);
-	            draw_text(xx+573,yy+66,string_hash_to_newline(string(ma_ar)));
-            
-	            x5+=string_width(string_hash_to_newline(ma_ar));
-	            draw_set_color(c_gray);
-	            if (ar_we1=1) then draw_set_color(c_gray);
-	            if (ar_we1=2) then draw_set_color(881503);
-	            draw_text(xx+573+x5,yy+66,string_hash_to_newline(string(ma_we1)));
-            
-	            x5+=string_width(string_hash_to_newline(ma_we1))+15;
-	            draw_set_color(c_gray);
-	            if (ar_we2=1) then draw_set_color(c_gray);
-	            if (ar_we2=2) then draw_set_color(881503);
-	            draw_text(xx+573+x5,yy+66,string_hash_to_newline(string(ma_we2)));
+
+						draw_set_color(c_gray);
+						if (ar_ge=1) then draw_set_color(c_gray);
```

Comment:
please rename vars, it's very hard to understand what is going on with names likes ar_ge, ar_mb, x5 etc

Reply:
These variables are actually shared with the marine equip logic. ge is the wargear slot and mb is the mobility slot. Effectively wep1, wep2, armour, wargear and mobility are the equipment 5 slots, although they are remapped onto wep1, wep2, wep3, upgrade and accessory for vehicles.

I'll update the x5 though, as I don't think it's relied upon anywhere else in the code.

Reply:
x5 now called "xoffset". x5 is referenced further down in the code, but it resets the variable as part of that section and doesn't have any direct relation to this part.

Reply:
Tidied up white space in this section and added comments to detail mapping between vehicle and marine equipment.